backend/scripts/test.bats

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bats
+
+@test "Passing wrong EDNA_API_LISTEN cla causes error" {
+  run edna-server --listen-addr "this_is_not_host_and_port"
+  assert_failure
+  assert_output --partial "EDNA_API_LISTEN"
+}
+
+@test "Setting wrong EDNA_API_LISTEN env causes error" {
+  EDNA_API_LISTEN="this_is_not_host_and_port"
+
+  run edna-server
+  assert_failure
+  assert_output --partial "EDNA_API_LISTEN"
+}
+
+
+@test "Passing wrong EDNA_DB_MAX_CONNECTIONS cla causes error" {
+  run edna-server --max-connections "this_is_not_a_number"
+  assert_failure
+  assert_output --partial "EDNA_DB_MAX_CONNECTIONS"
+}
+
+@test "Setting wrong EDNA_DB_MAX_CONNECTIONS env causes error" {
+  EDNA_DB_MAX_CONNECTIONS="this_is_not_a_number"
+
+  run edna-server
+  assert_failure
+  assert_output --partial "EDNA_DB_MAX_CONNECTIONS"
+}
+
+
+@test "Setting wrong POSTGRES_PORT env causes error" {
+  POSTGRES_PORT="this_is_not_a_number"
+
+  run edna-server
+  assert_failure
+  assert_output --partial "POSTGRES_PORT"
+}
+
+
+@test "Passing wrong EDNA_LOGGING_LEVEL cla causes error" {
+  run edna-server --logging wrong_logging_level
+  assert_failure
+  assert_output --partial "EDNA_LOGGING_LEVEL"
+}
+
+
+@test "Passing clas modifes config" {
+  run edna-server --listen-addr "127.0.0.1:80" --serve-docs --max-connections 1 --logging prod --dump-config
+  assert_success
+  assert_line --index 1 "api:"
+  assert_line --index 2 "  listen-addr: 127.0.0.1:80"
+  assert_line --index 3 "  serve-docs: true"
+  assert_line --index 4 "logging: LogProd"
```

Comment:
Actually it's pretty weird that we parse `Prod` but print `LogProd` (with `Log` prefix). Apparently there is a bug in
```
instance Aeson.ToJSON LoggingConfig where
  toEncoding = Aeson.genericToEncoding loggingConfigAesonOptions
```

it should define not only `toEncoding` but also `toJSON`.

Reply:
done